edgexfoundry/app-functions-sdk-go
# feat: Add capability to Publish to MessageBus when using non-MessageBus triggers

closes #1439







**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/app-functions-sdk-go/blob/main/.github/CONTRIBUTING.md

## PR Checklist
Please check if your PR fulfills the following requirements:

- [X] I have fully tested (add details below) this the new feature or bug fix (if not, why?) TBD
- [x] I have opened a PR for the related docs change (if not, why?) TBD
      https://github.com/edgexfoundry/edgex-docs/pull/1175

## Testing Instructions
1. run `make test`
1. Start edgex in no-secty
1. edit app-service-template to call the new functions for app service and app context in main.go
1. build app-service-template
1. subscribe to topics `edgex.event-xml` and `edgex.testing` (used for PublishWithTopic function) using rediscli
1. verify messages were sent

## New Dependency Instructions (If applicable)
